### DISCUSSION TITLE
Slider onUpdate event not firing when right and left button of keyboard is pressed

### DISCUSSION THREAD

**Discussion by muthubentley** [iTwin/iTwinUI#769] [General]
When slider is moved with mouse click or dragging the thumb of slider, thumb will moved to desired location and value in the slider gets updated and the onUpdate event gets fired. But when the slider is moved using right and left buttons in keyboard, thumb will be moved and value in the slider gets updated but onUpdate event does not fire. 
Refer to below sandbox to reproduce the behavior:
https://www.itwinjs.org/sandbox/MuthuKalamani/SliderWithKeyboardIssue
-Mouse click on slider --> alert message will be shown with the slider value 
-Move the slider knob --> alert message will be shown with the slider value 
-press right and left buttons in Keyboard --> alert message will not be shown

**Comment by mayank99** [comment]
There is an issue open for this already, can you leave a comment there? https://github.com/iTwin/iTwinUI-react/issues/398

**Reply by muthubentley** [reply]
sure will give the link of sandbox in the original issue
